src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/listeners/HMPPSAuditListener.kt

```diff
@@ -31,7 +31,7 @@ class HMPPSAuditListener(
     auditService.audit(auditEvent)
   }
 
-  @Entity
+  @Entity(name = "AuditEvent")
   @Table(name = "AuditEvent")
   data class AuditEvent(
```

Comment:
I got away with just `@Entity` on my last one of these - no table or name of entity required.  I was just wondering why you need them?

Reply:
I think because my entity is an inner class of the listener (maybe it shouldn't be?)
I also needed it on @Table because of that - as it was trying to use "listener.audit_event" for the table name.

Reply:
Ah... I wonder if that's because you put the entity next to the listener but I put the entity next to the repository...?